core/services/helper/main.py

```diff
@@ -52,6 +63,15 @@ def detect_service(port: int) -> ServiceInfo:
         if not info.valid:
             return info
 
+        # Check for service description metadata
+        try:
+            with requests.get(f"http://127.0.0.1:{port}/register_service", timeout=0.2) as response:
+                if response.status_code == http.HTTPStatus.OK:
+                    info.metadata = ServiceMetadata.parse_obj(response.json())
+        except Exception:
```

Comment:
shouldn't pylint catch this as a broad-exception?

we should at least log the exception here

Reply:
> shouldn't pylint catch this as a broad-exception?
> 
> we should at least log the exception here

We have it disabled: https://github.com/patrickelectric/companion-docker/blob/8ab788c945a1f656c2deedaaddf0096fb29fab21/pyproject.toml#L55

But I agree that we should log the issue.

Reply:
It should. Strange. And I can't find any `# pylint: ignore` blocks anywhere.